biotope/2018-06-05_biotopeVision.md

```diff
@@ -0,0 +1,20 @@
+# Biotope is a framework which enables you to develop scalabale, platform-agnostic frontend component ecosystems
+
+## It helps developers to:
+- easily develop ui-components following a unified standard
+- keep focus on developing components, the architecture and dev environment is provided by Biotope.
+- build platform-agnostic ui components which can be used in single page applications as well as in traditional server side rendered cms projects
+- make it easy to work with a team of many developers, all providing components to the library
+- easily ensure high quality standards for the whole library even when developed by multiple suppliers
+- build real projects and design system styleguides with a single code source
```

Comment:
Can we add "UI libraries" here? Or is this part of either "real projects" or "styleguides"?
Or do we want to say, that while building real components which can be used in real projects the styleguide can be created without much additional effort and therefore always kept in sync

Reply:
What we are doing with Biotope is building ui-libraries (ecosystems) all the components inside a library and addional design system information are showcased in a styleguide generated out of the UI library. 1-n real projects use 1-n components from the ui library. the important thing is that the styleguide and real projects are both using the exact same code from the ui library. thats a usp of biotope

Reply:
Ok. Now I get it.